news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
+subtitle: Here are some more details about it
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
+---
+
+## A new front-office theme is being built 
+
+For a long time, many community members pushed for a rework of the Classic front-office theme. The maintainer team is happy to announce that a new front-office theme is being built, and you can find [here](https://github.com/PrestaShop/theme-refacto).
+
+## Why such a decision?
+
+The PrestaShop community provided a [lot of feedback](https://github.com/PrestaShop/PrestaShop/issues/14533) and ideas, with suggestions like:
+
+-   Upgrade to Bootstrap 5
+-   Rework the structure inside  `src`  directory, which was  `_dev`  before.
+-   Remove the `core.js` file. Theme developers can't modify anything inside it or even remove it from their theme.
+-   Use [lazy loading](https://en.wikipedia.org/wiki/Lazy_loading).
+-   Choose, and use some CSS class naming strategy consistently, like [BEM](https://github.com/getbem/getbem.com/). This is more maintainable and easier to read.
+-   Use [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD) instead of microdata attributes.
+-   Add as many hooks as possible to extend the base theme.
+-   Separate the PrestaShop styling from the Bootstrap raw styling.
+-   Reduce the number of dependencies.
+-   Stop using jQuery in the theme. This would be the first step because modules use it anyway, so we can't remove it.
```

Comment:
```suggestion
-   Do not rely on jQuery. Note that jQuery will still be available as a lot of modules use it anyway.
```

It would be great if we could explain / reference this line because I guess a lot of people, not highly skilled in frontend, might say "why ? jQuery is very useful, why not use it?"

Reply:
Also maybe we should tell them what we used instead. Some people might believe we replaced jQuery by React 😄